assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -331,7 +331,7 @@ const baseSelectors = {
 				hasProductPostType
 					? KM_ANALYTICS_POPULAR_PRODUCTS
 					: KM_ANALYTICS_POPULAR_CONTENT,
-				KM_ANALYTICS_ENGAGED_TRAFFIC_SOURCE,
+				KM_ANALYTICS_ADSENSE_TOP_EARNING_CONTENT,
```

Comment:
@10upsimon, could you please fix jest test issues caused by this change?

Reply:
@eugene-manuilov this has been resolved in the latest commits, thank you.